src/mngt/ocf_mngt_cache.c

```diff
@@ -1542,13 +1609,15 @@ struct ocf_pipeline_properties _ocf_mngt_cache_attach_pipeline_properties = {
 	.priv_size = sizeof(struct ocf_cache_attach_context),
 	.finish = _ocf_mngt_cache_attach_finish,
 	.steps = {
+		OCF_PL_STEP(_ocf_mngt_copy_uuid_data),
 		OCF_PL_STEP(_ocf_mngt_attach_cache_device),
-		OCF_PL_STEP(_ocf_mngt_attach_load_properties),
+		OCF_PL_STEP(_ocf_mngt_init_properties),
+		OCF_PL_STEP(_ocf_mngt_attach_properties),
```

Comment:
load in "_ocf_mngt_load_properties" reffers to loading properties rather than OCF cache load. To avoid ambiguity I would rename "load" to "read", so here we would have "_ocf_mng_attach_read_properties".